turbo.json

```diff
@@ -28,25 +28,44 @@
     "test": {
       "dependsOn": [
         "^build"
-      ],
-      "outputs": []
+      ]
     },
-    "lint": {
-      "outputs": []
+    "lint": {},
+    "start": {
+      "outputs": [
+        "dist/**",
+        "build/**"
+      ]
+    },
+    "size": {
+      "outputs": [
+        "dist/**",
+        "build/**"
+      ]
     },
-    "start": {},
-    "size": {},
     "dev": {
       "cache": false
     },
     "dev:server": {
       "cache": false
     },
     "@faststore/api#dev:graphql": {
+      "outputs": [
+        "dist/**",
+        "build/**"
```

Comment:
It doesn't output to build, neither does `faststore/api#build:cjs` or `@faststore/api#build:esm`

Reply:
this was automatically generated by codemode.

But I think it's correct, don't you? doesn't dev create these folders?

Reply:
No, it only creates `dist` 🤔

Reply:
I'll check it

Reply:
done, @icazevedo can u review again?